### ISSUE TITLE
MCS Tracking

### ISSUE THREAD

**Issue by VTIITI** [alramirezca/ATRACKCS#1]
Hi 
Thanks for developing such a very useful package. I am trying to detect the MCS over the Kerala domain in India using this module. I tried and got the MCS and tracks but could not figure out why I am getting it over the South American region. I changed the UTM zone code for Kerala. 

can you suggest the possible mistake I am making?
**UTM_LOCAL_ZONE = 32644
UTC_LOCAL_HOUR = 5.5  
UTC_LOCAL_SIGN = "plus"  
TB = 215  
AREA_TB = 1000
THRESHOLD_OVERLAPPING_P = 15
LOCATION_FOLIUM = [10, 76]  
MIN_DURATION = 3**

Do I need to make any other changes?
![image](https://github.com/alramirezca/ATRACKCS/assets/111271607/1858267a-204b-40b1-b527-e76e4e78c788)

**Comment by alramirezca** [comment]
Hi,

Thank you for your positive feedback, and I'm delighted to know that you're finding the package valuable. It appears that you've encountered unexpected results while attempting to detect MCS over the Kerala domain in India. Despite your adjustments, you've mentioned that the package is still yielding data pertaining to the South American region.

I've noticed that the results you're obtaining seem to align with the examples provided in the package's repository. Have you ensured that the input files you've downloaded align with the specific requirements of your domain? If you've done so, it would be helpful if you could share some brightness temperature netcdf files related to your domain, which would aid in validating the results.

It's possible that you might be assuming the ability to track MCS in any domain based on the examples provided in the repository. However, it's important to note that the netcdf files provided in the example exclusively contain data relevant to the South American region, and are therefore specifically intended for tracking MCS in this region. I would recommend exploring the documentation in the repository that covers the algorithm input data to ensure you are downloading netcdf files that pertain to your region and the desired time frame accurately.

I remain attentive to your response.

**Comment by vt-tyagi** [comment]
Thanks for your prompt reply. 

Sorry I forgot to mention that I have downloaded the subsetted data [lat: 0-30, lon: 65-105] Tb and P both for the Indian region. For now, I am trying the algo with Tb only. 

I am adding the test data that I am using. 

Once again thanks for helping and actively responding.

[test.zip](https://github.com/alramirezca/ATRACKCS/files/12895627/test.zip)

**Comment by alramirezca** [comment]
Hi.

I successfully ran the algorithm using the files you provided, and encountered no issues. Here are the results after adjusting the TB threshold to obtain more MCS in the input region. Once you've tracked all the MCS, you must modify the input path in the funcs.readTRACKS(). I assume you're using the same resume_Tb as in the example, which explains the results you achieved in South America. I have attached the notebook and the results obtained with the shared files for your reference.

![image](https://github.com/alramirezca/ATRACKCS/assets/65802031/a30fb861-bbb5-4695-a978-4145df60a381)
[issue_atrackcs_kerala.zip](https://github.com/alramirezca/ATRACKCS/files/12923251/issue_atrackcs_kerala.zip)
